backend/apps/loginapp/loginappframework/apis/register.js

```diff
@@ -125,6 +140,19 @@ exports.addNewUserListener = (modulePath, functionName) => {
 
 exports.REASONS = REASONS;
 
+async function _emailUnknownEmail(id, name, org, lang, bgc) {
+	const cryptID = crypt.encrypt(id), cryptTime = crypt.encrypt(utils.getUnixEpoch().toString()), 
+        action_url = APP_CONSTANTS.CONF.base_url + Buffer.from(`${APP_CONSTANTS.CONF.approve_url}?e=${encodeURIComponent(cryptID)}&t=${encodeURIComponent(cryptTime)}${bgc?`&bgc=${encodeURIComponent(bgc)}`:""}`).toString("base64"),
+		button_code_pre = mustache.render(emailTemplate.button_code_pre, {action_url}), 
+		button_code_post = mustache.render(emailTemplate.button_code_post, {action_url})
+	
+	const email_title = mustache.render(emailTemplate[`${lang||"en"}_unknown_title`], {name, org}),
+		email_html = mustache.render(emailTemplate[`${lang||"en"}_unknown_html`], {id, name, org, button_code_pre, button_code_post}),
+		email_text = mustache.render(emailTemplate[`${lang||"en"}_unknown_text`], {id, name, org, button_code_pre, button_code_post});
+	
+	return await mailer.email(APP_CONSTANTS.CONF.tekmonks_admin_email, email_title, email_html, email_text);
```

Comment:
Hi @TekMonksGitHub, 

I guess the `approveEmail` API that I made is much more similar to `verifyEmail` than `approveUser`. Is it okay if that's the API that I should update instead of `approveUser`?